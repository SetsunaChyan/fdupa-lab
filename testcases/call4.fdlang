function main(){
    a = 20;
    call callee1(a);
    call callee2(a);
}

function callee1(m){
    m = m + 1;
    check_interval(m, 21, 21);
}

function callee2(m){
    m = m - 1;
    n = 5;
    check_interval(m, 21, 21);
    call callee3(m,n);
}

function callee3(x,y){
    x = x + 5;
    if(y < 9){
         check_interval(x, 24, 24);
    }else{
        check_interval(y, 20, 20);
    }
}