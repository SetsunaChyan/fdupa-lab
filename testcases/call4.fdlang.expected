Line 9: YES; m in [21, 21]
Line 15: NO; n in [19, 19]
Line 22: YES; x in [24, 24]
Line 24: Unreachable